File: Seccion 12 - Punteros en C++/Seccion_12_Python/01_ejercicio_01.py

def main():
    
    numero: int
    
    numero = int(input('Digite un número: '))
    
    if numero % 2 == 0:
        print('El número {} es par.'.format(numero))
    else:
        print('El número {} es impar.'.format(numero))

if __name__ == '__main__':
    main()

File: Seccion 12 - Punteros en C++/Seccion_12_Dart/01_ejercicio_01.dart

import 'dart:io';

void main() {

  int numero;

  stdout.write('Digite un número: ');
  numero = int.parse(stdin.readLineSync());

  if (numero % 2 == 0) {
    print('El número $numero es par.');
  } else {
    print('El número $numero es impar.');
  }
}